cmd/state/internal/cmdtree/cmdtree.go

```diff
@@ -113,6 +113,68 @@ func New(prime *primer.Values, args ...string) *CmdTree {
 
 	applyLegacyChildren(stateCmd, globals)
 
+	groups := []captain.CommandGroup{
+		{
+			Message: "Package Management:",
+			Commands: []*captain.Command{
+				stateCmd.FindSafe([]string{"packages", "add"}),
+				stateCmd.FindSafe([]string{"packages", "remove"}),
+				stateCmd.FindSafe([]string{"packages", "import"}),
+				stateCmd.FindSafe([]string{"packages", "search"}),
+				stateCmd.FindSafe([]string{"packages"}),
+			},
+		},
+		{
+			Message: "Environment Management:",
+			Commands: []*captain.Command{
+				stateCmd.FindSafe([]string{"activate"}),
+				stateCmd.FindSafe([]string{"init"}),
+				stateCmd.FindSafe([]string{"clean"}),
+				stateCmd.FindSafe([]string{"deploy"}),
+			},
+		},
+		{
+			Message: "Platform Tools:",
+			Commands: []*captain.Command{
+				stateCmd.FindSafe([]string{"auth"}),
+				stateCmd.FindSafe([]string{"invite"}),
+				stateCmd.FindSafe([]string{"organizations"}),
+				stateCmd.FindSafe([]string{"platforms"}),
+				stateCmd.FindSafe([]string{"projects"}),
+				stateCmd.FindSafe([]string{"show"}),
+			},
+		},
+		{
+			Message: "Automation:",
+			Commands: []*captain.Command{
+				stateCmd.FindSafe([]string{"events"}),
+				stateCmd.FindSafe([]string{"scripts"}),
+			},
+		},
+		{
+			Message: "Version Control:",
+			Commands: []*captain.Command{
+				stateCmd.FindSafe([]string{"fork"}),
+				stateCmd.FindSafe([]string{"pull"}),
+				stateCmd.FindSafe([]string{"push"}),
+				stateCmd.FindSafe([]string{"history"}),
+			},
+		},
+		{
+			Message: "Utilities:",
+			Commands: []*captain.Command{
+				stateCmd.FindSafe([]string{"update"}),
+				stateCmd.FindSafe([]string{"export"}),
+			},
+		},
+	}
+
+	templater := captain.Templater{
+		Cmd:           stateCmd,
+		CommandGroups: groups,
+	}
+	stateCmd.SetUsageFunc(templater.RootCmdUsageFunc())
```

Comment:
cmdtree is for registering command structure, captain is for representing the commands. Setting the template should happen in captain.

Reply:
We were already setting the template in this file though I'm not sure why. I've removed it.